src/main/java/GameFrame.java

```diff
@@ -2,26 +2,62 @@
 import java.awt.*;
 
 public class GameFrame extends JFrame {
-    JButton nextButton;
+    private JButton nextButton;
+    private JButton clearButton;
+    private JPanel buttonPanel;
+    private Grid grid;
+    private GridView gridView;
+    private Dimension buttonMaxSize;
 
     public GameFrame(Grid grid,
                 GridView gridView,
                 GameMouseClickListener mouseClickListener) {
         super();
 
-        setSize(815, 480);
+        this.grid = grid;
+        this.gridView = gridView;
 
+        setSize(800, 400);
```

Comment:
uh - oh, that doesn't look right. Was the consensus that we change it to `setSize(815, 480)`?